src/libcore/convert.rs

```diff
@@ -220,21 +220,20 @@ pub trait AsMut<T: ?Sized> {
 ///
 /// # Implementing [`Into`] for conversions to external types
 ///
-/// If the destination type is not part of the current crate
-/// then you can't implement [`From`] directly.
+/// Prior to Rust 1.40, if the destination type was not part of the current crate
```

Comment:
I think we should strengthen this -- unless we have a good example of when you *do* need to implement `Into`, we should probably start by telling people to **avoid** implementing `Into` directly, and then continue with this content?

Reply:
This should say 1.41 instead?